wp-includes/js/offline-page-handling.template.js

```diff
@@ -0,0 +1,26 @@
+/* global OFFLINE_PAGE_URL */
+( () => {
+
+	// Add custom offline / error response serving.
+	const networkFirstHandler = wp.serviceWorker.strategies.networkFirst( {
```

Comment:
I think we need to re-use `precache` as the `cacheName` for this so that it will use what was already cached.